feature-libs/storefinder/core/store/selectors/find-stores.selectors.ts

```diff
@@ -28,3 +28,10 @@ export const getStoresLoading: MemoizedSelector<
 > = createSelector(getFindStoresState, (state) =>
   StateUtils.loaderLoadingSelector(state)
 );
+
+export const getStoresSucess: MemoizedSelector<
```

Comment:
typo. 

if you have not already, you might want to use https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker